src/scripts/components/LoginPage.tsx

```diff
@@ -133,29 +131,112 @@ function HeaderLinks ({ loggedIn }: Pick<LoginState, 'loggedIn'>) {
+      </li>
       <li>
         <Link href="settings.html?tab=account">
           Account
         </Link>
       </li>
       <li>
-        <Link href="settings.html">
-          Settings
+        <Link href="https://support.toggl.com/browser-extensions">
+          User Guide
         </Link>
       </li>
     </Links>
   );
 }
 
+function QuickstartGuide () {
+  return (
+    <Content>
+      <Row>
+        <Heading>Welcome to Toggl Button!</Heading>
+        <Subheading>Track time from anywhere on the web.</Subheading>
+      </Row>
+      <Row>
+        <a href="#quickstart">
+          <Button>Get started</Button>
+        </a>
+      </Row>
+      <Guide id="quickstart">
+        <GuideHeading>
+          <h2>Quick start</h2>
+          <p>Toggl Button's top features and how to use them:</p>
+        </GuideHeading>
+        <div css={[section, right, salmon]}>
+          <div>
+            <h5>Easy & fast</h5>
+            <h3>Create time entries</h3>
+            <p>Just click the "Play" icon to start recording your first time entry!</p>
+            <p>When you're done with your task, click on the "Stop" icon to save your entry.</p>
+            <p>The Toggl Button popup also shows a list of all your recent time entries*.</p>
+            <p>You can also enable useful time tracking features like idle detection and reminders from the Settings page.</p>
+            <Link href="settings.html"><Button>Settings</Button></Link>
+            <Link href="https://toggl.com/app" target="_blank"><p className="small">*To see older time entries, or run more complex reports, check out our web app!</p></Link>
+          </div>
+          <img src="../images/button-create.gif" />
+        </div>
+        <div css={[section, left, gold]}>
+          <div>
+            <h5>Supercharge your tools</h5>
+            <h3>Use integrations</h3>
+            <p>Add a Toggl Button to 130+ services for super fast time tracking!</p>
+            <p>Once enabled, just click on the Toggl Button icon to start a time entry. Toggl Button will automatically fill in the description for you.</p>
+            <p>If Toggl Button can detect a project, <em>and</em> there's project with the same name in Toggl, it'll be selected too!</p>
+            <p>You can stop entries by clicking the same icon again, or move straight onto another task by clicking the new task's icon.</p>
+            <p>Check out the Integrations page now to see if your tool is supported!</p>
+            <Link href="settings.html?tab=integrations"><Button>Integrations</Button></Link>
+          </div>
+          <img src="../images/button-integrations.gif" />
+        </div>
+        <div css={[section, right, green]}>
+          <div>
+            <h5>Increase your productivity</h5>
+            <h3>Pomodoro mode</h3>
+            <p>Need help managing your time?</p>
+            <p>Pomodoro is a technique to help you focus on a task, by working hard for short intervals then taking a break.</p>
```

Comment:
```suggestion
            <p>Pomodoro® is a technique to help you focus on a task, by working hard for short intervals then taking a break.</p>
```

Add `®` for [Pomodoro](https://francescocirillo.com/pages/pomodoro-technique)